config/locales/en.yml

```diff
@@ -32,4 +32,6 @@
 en:
   invalid_name: 'Please enter a valid name'
   player_successful_updated: 'Player successfully updated.'
-  player_successful_deletion: 'Player successfully deleted.'
\ No newline at end of file
+  player_successful_deletion: 'Player successfully deleted.'
+  deck_successful_deletion: 'Deck successfully deleted.'
+  deck_successful_updated: 'Deck successfully updated'
```

Comment:
Eventually those could be abstracted, no need to do that right now.
```yml
successful_deletion: 'successfully deleted'
deck: 'deck'
player: 'player'
```
and assemble the message in the controllers.
```rb
def delete_successful_message(record)
  message = "#{t(record).capitalize} #{t(:successful_deletion)}."
end
```
again don't do this right now just throwing out ideas.